src/framework/components/collision/component.js

```diff
@@ -123,6 +125,18 @@ class CollisionComponent extends Component {
         }
     }
 
+    onSetLinearOffset(name, oldValue, newValue) {
+        if (this.data.initialized) {
+            this.system.recreatePhysicalShapes(this);
+        }
+    }
+
+    onSetAngularOffset(name, oldValue, newValue) {
+        if (this.data.initialized) {
+            this.system.recreatePhysicalShapes(this);
+        }
+    }
```

Comment:
These could be the same function. 😄  (e.g. rename to `onSetOffset`).

Reply:
Good point. My original functions had other stuff there. After removing them, they no longer have a difference. Will use a single method.